yw_website/yw_website/apps/website/templates/includeable/tagarray.html

```diff
@@ -0,0 +1,14 @@
+{% load static %}
+{% get_static_prefix as STATIC_PREFIX %}
+<link rel="stylesheet" type="text/css" href="{{ STATIC_PREFIX }}css/bulma.css">
+
+<div class="tags has-addons">
+{% for tag in tags %}
+  <div class="control">
+    <div class="tags has-addons">
+      <a href="{% url 'home' %}?q={{ tag.tag.title }}" class="tag is-link">{{tag.tag.title}}{% if not forloop.last %}{% endif %}</a>
```

Comment:
not sure why you have `{% if not forloop.last %}{% endif %}`

Reply:
Oops relic of how I was adding commas

Reply:
lol